models/demos/t3000/falcon40b/tt/model_config.py

```diff
@@ -727,7 +727,7 @@ def get_prefill_model_config(model_config_str, input_shape, num_devices):
     model_config["KV_CACHE_MEMCFG"] = DRAM_MEMCFG
     model_config["KV_CACHE_DTYPE"] = BFP8_DTYPE
 
-    model_config["ATTN_MASK_DTYPE"] = BFP4_DTYPE
+    # model_config["ATTN_MASK_DTYPE"] = BFP8_DTYPE  # SPDA op requires all inputs to have the same dtype and only supposts bfp8 and fp16
```

Comment:
I personally prefer to keep this line commented in and set the dtype explicitly, because there's a reason why we need it in bfp8 and thus its clear that it's bfp8 and also that we need it to be bfp8.